src/components/NavBar/LightDarkSlider.tsx

```diff
@@ -0,0 +1,45 @@
+import { Switch } from "@mui/material";
+import styled from "styled-components";
+
+const LightDarkSlider = styled(Switch)`
+    width: 62px;
+    height: 34px;
+    padding: 7px;
+    & .MuiSwitch-switchBase {
+        margin: 1px;
+        padding: 0;
+        transform: translateX(6px);
+        &.Mui-checked {
+            transform: translateX(22px);
+            & .MuiSwitch-thumb:before {
+                background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" height="20" width="20" viewBox="0 0 20 20"><path fill="%23FFFFFF" d="M4.2 2.5l-.7 1.8-1.8.7 1.8.7.7 1.8.6-1.8L6.7 5l-1.9-.7-.6-1.8zm15 8.3a6.7 6.7 0 11-6.6-6.6 5.8 5.8 0 006.6 6.6z"/></svg>');
+            }
+            & + .MuiSwitch-track {
+                opacity: 1;
```

Comment:
is opacity 1 not default?

Reply:
No, the MUI slider by default has different opacities for the switch track when it is in 'on position' and 'off position' so `opacity: 1;` is necessary